### PR TITLE
feat: Update container monitoring addon for may omsagent release 

### PR DESCRIPTION
**Pull request by rashmichandrashekar**
Updated api versions for cluster role and cluster role binding
Adding permissions for nodes/proxy
Updating agent image version to ciprod05262020

### CODE REVIEW

File: parts/k8s/addons/container-monitoring.yaml [hunk 1]
```diff
@@ -22,15 +22,15 @@ metadata:
     addonmanager.kubernetes.io/mode: Reconcile
 ---
 kind: ClusterRole
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
```
Comment (jackfrancis) [inline parts/k8s/addons/container-monitoring.yaml:25]: Does this change work across all currently supported Kubernetes versions? (1.15-1.19)

File: pkg/api/addons_test.go [hunk 2]
```diff
@@ -1564,7 +1564,7 @@ func TestSetAddonsConfig(t *testing.T) {
 					Enabled: to.BoolPtr(true),
 					Config: map[string]string{
 						"omsAgentVersion":       "1.10.0.1",
-						"dockerProviderVersion": "8.0.0-3",
+						"dockerProviderVersion": "10.0.0-1",
```
Comment (jackfrancis) [inline pkg/api/addons_test.go:1567]: This changed in the unit test, but we didn't change the actual addon default, so the UT is failing.

Comment (rashmichandrashekar) [reply pkg/api/addons_test.go:1567]: good catch.. fixed it

File: parts/k8s/addons/container-monitoring.yaml (hunk 1 above)
Comment (rashmichandrashekar) [reply parts/k8s/addons/container-monitoring.yaml:25]: Yes this works.

**Pull request merged**